src/pretix/presale/templates/pretixpresale/event/checkout_addons.html

```diff
@@ -45,15 +48,277 @@ <h4 class="panel-title">
                                             {% plural %}
                                                 You need to choose {{ min_count }} options from this category.
                                             {% endblocktrans %}
-                                        {% elif c.min_count == 0 and c.max_count >= c.form.fields|length %}
+                                        {% elif c.min_count == 0 and c.max_count >= c.items|length and not c.multi_allowed %}
+                                        {% elif c.min_count == 0 %}
+                                            {% blocktrans trimmed with max_count=c.max_count %}
+                                                You can choose up to {{ max_count }} options from this category.
```

Comment:
This is only displayed, when the Multi-AddOn-selection is enabled.

In the "classic" step, it should probably read "You can select up to {{ max(max_count, len(items)) }} options" to not confuse people (for example: Can select 0 to 10 items, but only 3 items are available to choose from).